src/test/java/com/microsoft/sqlserver/jdbc/socketfactory/CustomSocketFactoryTest.java

```diff
@@ -91,7 +91,6 @@ public void testCustomSocketFactory() throws Exception {
         try (Connection con = PrepUtil.getConnection(url)) {
             Assert.assertTrue(con != null);
         }
-        Assert.assertEquals("The custom socket factory should have been used once", 1, dummyLog.size());
```

Comment:
is this a valid test when it's not AzureDB? If so, add xAzureSQLDB tag instead?

Reply:
It's not about whether the test server is against AzureDB, it has to do with the environment in which the test is performed. I don't think we should be adding a tag for this, and the number of times the custom socket factory being called is a trivial test.